calico/getting-started/windows-calico/quickstart.md

```diff
@@ -31,9 +31,12 @@ For Linux control nodes using {{site.prodname}} networking, strict affinity must
 This is required to prevent Linux nodes from borrowing IP addresses from Windows nodes:
 
 ```bash
-calicoctl ipam configure --strictaffinity=true
+kubectl patch ipamconfigurations default --type merge --patch='{"spec": {"strictAffinity": true}}'
 ```
 
+> **Note**: If the above command failed to find ipamconfigurations resource, you need to install Calico API server. Please refer to [installing the Calico API server]({{site.baseurl}}/maintenance/install-apiserver).
```

Comment:
nit:

> If the above command failed to find the ipamconfigurations resource, you need to install the Calico API server.

Reply:
Do we want to add this note in the other Windows docs pages too?

Reply:
We can but I just feel a bit redundant. The quick start guard is the page most of the users will land.